ext/CEGUI/CMakeLists.txt

```diff
@@ -1,19 +1,33 @@
 include(../../cmake/common.cmake)
 update_git_submodule(./stb)
 
-
+# to remove later, only adding my source
 set(IRR_EXT_CEGUI_H
   ExtCEGUI.h
 	CEGUIOpenGLState.h
 	CEGUIOpenGLClip.h
 	Helpers.h
+	"IrrlichtBaW/include/ExtGeometryBuffer.h"
+	"IrrlichtBaW/include/ExtImageCodec.h"
+	"IrrlichtBaW/include/ExtRenderer.h"
+	"IrrlichtBaW/include/ExtRenderTarget.h"
+	"IrrlichtBaW/include/ExtShaders.h"
+	"IrrlichtBaW/include/ExtTexture.h"
+	"IrrlichtBaW/include/ExtTextureTarget.h"
+	"IrrlichtBaW/include/ExtViewportTarget.h"
 )
 
 set(IRR_EXT_CEGUI_SRC
   ExtCEGUI.cpp
 	CEGUIOpenGLState.cpp
 	CEGUIOpenGLClip.cpp
 	Helpers.cpp
+	"IrrlichtBaW/src/ExtGeometryBuffer.cpp"
+	"IrrlichtBaW/src/ExtImageCodec.cpp"
+	"IrrlichtBaW/src/ExtRenderer.cpp"
+	"IrrlichtBaW/src/ExtTexture.cpp"
+	"IrrlichtBaW/src/ExtTextureTarget.cpp"
+	"IrrlichtBaW/src/ExtViewportTarget.cpp"
```

Comment:
no need for the `IrrlichtBaW` in the path

also don't add `include` and `src` to paths in `ext` just keep it all in one folder